src/style.css

```diff
@@ -1,3 +1,91 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+header {
+  width: 100%;
+  background-color: #fff;
+}
+
+.conteiner {
```

Comment:
There is a typo in the class name 'conteiner'. It should be 'container'. This should be consistent with the HTML to ensure the styles are applied correctly.